src/v2/components/ui/TextField.tsx

```diff
@@ -60,13 +60,44 @@ const Label = styled("label", {
   },
 });
 
+const Side = styled("span", {
+  position: "absolute",
+  top: "50%",
+  right: 20,
```

Comment:
It seems just replacing in this PR but... how about specifying the unit for this value?

Reply:
The unit-less values are used quite frequently throughout the source code. Let's discuss this on a separate issue.